src/interface.jl

```diff
@@ -230,6 +230,22 @@ transform!(::typeof(identity), x, y) = copy!(y, x)
 logabsdetjac(::typeof(identity), x) = zero(eltype(x))
 logabsdetjac!(::typeof(identity), x, logjac) = logjac
 
+###################
+# Other utilities #
+###################
+"""
+    is_monotonically_increasing(f)
+
+Returns `true` if `f` is monotonically increasing.
+"""
+is_monotonically_increasing(f) = false
+is_monotonically_increasing(::typeof(identity)) = true
+function is_monotonically_increasing(cf::ComposedFunction)
+    return is_monotonically_increasing(cf.inner) && is_monotonically_increasing(cf.outer)
+end
+is_monotonically_increasing(::typeof(exp)) = true
```

Comment:
```suggestion
is_monotonically_increasing(::typeof(exp)) = true
is_monotonically_increasing(::typeof(log)) = true
is_monotonically_increasing(binv::Inverse) = is_monotonically_increasing(inverse(b))
```

Reply:
Since this is an interface function, would it be better to place these methods in the files where the corresponding bijectors are implemented? Also, I think we can mark this as true for `Logit`, `LeakyReLu`, `Scale` (when scale is positive), `Shift`, and `TruncatedBijector`.

Reply:
Agreed:)